patches/tModLoader/Terraria/ModLoader/ModPylon.cs

```diff
@@ -254,6 +254,7 @@ public override bool TileFrame(int i, int j, ref bool resetFrame, ref bool noBre
 			return true;
 		}
 
+		WorldGen.KillTile_DropItems(i, j, tileSafely, includeLargeObjectDrops: true, includeAllModdedLargeObjectDrops: true); // include all drops.
```

Comment:
Do we need to recommend people do similar if they had tiles with a similar implementation?

Reply:
I don't think any modders will be using code like this. ModPylon is strange in how it does things.